extensions/wikia/UserLogin/js/spec/UserBaseAjaxForm.spec.js

```diff
@@ -0,0 +1,54 @@
+/* global UserBaseAjaxForm */
+
+describe('Base class for User Ajax Forms', function () {
+	'use strict';
+
+	var instance, $el, options,
+		proto = UserBaseAjaxForm.prototype,
+		$userNameInput = $('<input type="text" name="username">'),
```

Comment:
I would rather wrote it like that:
```javascript
$('<input>', {
  type: 'text',
  name: 'username'
})
```
It looks like much more complicated. And It has also the advantages: No likely that one could tell "Use some template for that markup, mate" :), and IMO it is more convenient to maintenance later.